common/entity/ParticipationCertificate.ts

```diff
@@ -58,4 +58,19 @@ export class ParticipationCertificate {
 
     @Column({ default: false })
     ongoingLessons: boolean;
+
+    @Column({ default: "manual" })
+    state: "manual" | "awaiting-approval" | "approved";
+
+    @Column({ nullable: true, type: "bytea", select: false })
```

Comment:
If we’re storing the image data as base64 (this is how I understand the rest of the code), why do we not just use a simple `text` column in the database instead? Is `bytea` performing better than _text_?

Reply:
No, I think performance wise they are more or less equal. That was based on "In short, binary strings are appropriate for storing data that the programmer thinks of as "raw bytes", whereas character strings are appropriate for storing text." in the Postgres Docs. But I guess TEXT is also fine.